JS.md

```diff
@@ -62,51 +62,87 @@ The following intrinsic objects are added:
 
 #### `WebAssembly.validate`
 
-The `validate` function has the signature:
+The `validate` function is overloaded based on types of its arguments.
+If none of the following overloads match, then the returned `Promise` is
+[rejected](http://tc39.github.io/ecma262/#sec-rejectpromise)
+with a [`TypeError`](https://tc39.github.io/ecma262/#sec-native-error-types-used-in-this-standard-typeerror).
 
 ```
 Boolean validate(BufferSource bytes)
+
+Boolean validate(Response source)
```

Comment:
Hmm, since this is a synchronous method (it returns a Boolean), how can it take a Response? Should it instead create a Promise<Response>? In that case, I think it should not be an overload, but a separate function.